packages/app/src/app/mobile-placeholder.tsx

```diff
@@ -0,0 +1,25 @@
+'use client'
+
+import React from 'react'
+import { Container, Logo, Wrapper } from '../../components/mobile-placeholder.tsx/styled'
+
+function MobilePlaceholder() {
+  return (
+    <Wrapper>
+      <img src="/img/bg.png" alt="" className="bg" />
+      <Container>
+        <Logo src="/img/logo.svg" alt="" />
+
+        <div>
+          <div className="picture">
+            <img src="/img/mobile-placeholder-sm.jpg" alt="" className="sm" />
+            <img src="/img/mobile-placeholder-md.jpg" alt="" className="md" />
+          </div>
+          <div>Waiting for you on the desktop version</div>
+        </div>
+      </Container>
+    </Wrapper>
+  )
+}
+
+export default MobilePlaceholder
```

Comment:
Let's not use default exports for components for simplify consistent renaming and finding usages with IDE.

Reply:
Done